### DISCUSSION TITLE
Does Open metadata generates the metadata in json or flat file format?

### DISCUSSION THREAD

**Discussion by LingamAnitha** [open-metadata/OpenMetadata#7965] [Q&A] [answered]
Does Open metadata generates the metadata in json or flat file format?

**Comment by ayush-shah** [comment, marked as answer]
The storage of metadata is done in mysql under json file format.
cc: @ulixius9 

@LingamAnitha, it will be easier to support you from Slack, [here is the link](https://slack.open-metadata.org/) to our community.
